src/model/ServerModel.java

```diff
@@ -0,0 +1,19 @@
+package model;
+
+public class ServerModel {
+	
+	//I am the god of this game, despite this I am dumb.
+	//I do not know anything about any other part of the architecture.
+	//My methods are called by the ServerController who owns me. I act accordingly.
+	
+	//in theory, I will hold:
+		//- The master copy of the board
+		//- All character and player data
+		//- Basicly everything, I am god.

```

Comment:
yeah go for it, I can always fix it if something breaks

On Sun, Feb 15, 2015 at 1:54 PM, ChantalForget <notifications@github.com>
wrote:

> In src/model/ServerModel.java
> <https://github.com/sstone93/MAGIC/pull/17#discussion_r24724838>:
>
> > @@ -0,0 +1,19 @@
> > +package model;
> > +
> > +public class ServerModel {
> > +	
> > +	//I am the god of this game, despite this I am dumb.
> > +	//I do not know anything about any other part of the architecture.
> > +	//My methods are called by the ServerController who owns me. I act accordingly.
> > +	
> > +	//in theory, I will hold:
> > +		//- The master copy of the board
> > +		//- All character and player data
> > +		//- Basicly everything, I am god.
>
> [image: :godmode:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sstone93/MAGIC/pull/17/files#r24724838>.
>



-- 
Nicholas Laws